.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2.1
 executors:
   golang:
     docker:
-      - image: circleci/golang:1.15
+      - image: circleci/golang:1.16
```

Comment:
Here and in go.mod: I think we can upgrade safely but without further tests I would like to keep this out of this PR